src/common/header/header.css

```diff
@@ -244,6 +244,22 @@
   fill: var(--color-neutral-10);
 }
 
+.header-links > li > .app-header-btn:hover .share-icon {
+  fill: #a83f39;
+}
+
+.header-links > li > .app-header-btn:hover .github-icon {
+  fill: #2EAADA;
```

Comment:
The blue color that I adopted for the github icon is because ther is another github icon on the landing page that has blue color in it.
![image](https://user-images.githubusercontent.com/44437268/198831455-333a521d-bb1f-4f59-988c-0cfa7dbda200.png)

Reply:
yes, but the background color doesn't fit right